commands/add.go

```diff
@@ -15,7 +15,7 @@ const (
 )
 
 func init() {
-	RootCmd.AddCommand(&cobra.Command{
+	rootCmd.AddCommand(&cobra.Command{
```

Comment:
Well, this change is not related to the PR. Please try to not combine different changes in a single PR :)